Applications/CLI/CMakeLists.txt

```diff
@@ -34,3 +34,8 @@ cpack_add_component(ogs_cli
     DESCRIPTION "The command line interface for OpenGeoSys."
     GROUP Applications
 )
+
+if(NOT APPLE)
```

Comment:
Does it fail on Mac or is it unnecessary on Mac? (Maybe add a comment in the cmake file?)